t/27-make-update-deps.t

```diff
@@ -16,12 +16,18 @@
 
 use strict;
 use warnings;
+use Module::Load::Conditional qw(can_load);
 use Test::More;
 use Test::Warnings;
 use FindBin '$Bin';
 
 if (not -e "$Bin/../.git") {
-    pass("Skipping all tests, not in a git repository");
+    pass('Skipping all tests, not in a git repository');
+    done_testing;
+    exit;
+}
+if (!can_load(modules => {'YAML::PP' => undef})) {
+    pass('Skipping all tests, YAML::PP is not installed');
```

Comment:
Not sure, why should it be a good idea to pretend that everything was fine here?

Reply:
Yes, I better just remove the file.